docs/OracleDatastream-cdcSource.md

```diff
@@ -53,49 +53,49 @@ See [Configure your source Oracle database](https://cloud.google.com/datastream/
 Plugin properties
 -----------
 
-**Use an existing Datastream stream:** Whether to let this plugin create a new Datastream stream or use an existing Datastream stream. It's recommended that you only have one Datastream stream per Oracle Database for performance concerns.
+**Use an existing Datastream stream**: Whether to let this plugin create a new Datastream stream or use an existing Datastream stream. It's recommended that you only have one Datastream stream per Oracle Database for performance concerns.
```

Comment:
Does this mean we recommend setting this property to Yes to use existing datastream for best performance?